rotkehlchen/db/optimismtx.py

```diff
@@ -21,10 +21,13 @@ class DBOptimismTx(DBEvmTx):
     def add_evm_transactions(
             self,
             write_cursor: 'DBCursor',
-            evm_transactions: list[OptimismTransaction],  # type: ignore[override]
+            evm_transactions: list[L2WithL1FeesTransaction],  # type: ignore[override]
```

Comment:
Either in this or another PR this class can be renamed as also mentioned above before.